moped-editor/src/views/staff/helpers.js

```diff
@@ -68,8 +68,9 @@ export function useUserApi() {
 const errorsToTranslate = {
   "value does not match regex '^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+$'":
     "not a valid email format",
-  "value does not match regex '^[a-zA-Z0-9_-!@%^*~?.:&*()[]$]*$'":
-    "password must be at least 12 characters long, it must include at least one lowercase letter, one uppercase letter, one number, and one special character: _-!@%^~?.:&()[]$",
+  "value does not match regex '^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{12,}$'":
+    "password must be at least 12 characters long, include at least one lowercase letter, one uppercase letter, one number, and one special character: _-!@%^~?.:&()[]$",
+  "min length is 8": "password is too short",
```

Comment:
this error "min length is 8" comes from cognito. I tried using a blank string skip showing this error, but the formatApiErrors function `errorsArray.map((error) => errorsToTranslate[error] || error).join(", ")` evaluates the empty string as false, which shows the "min length is 8" error anyways. 

I welcome edits on what to show instead of this error

Reply:
I think this text is great and leads us in the right direction w/o the confusion of seeing two lengths requirements. 🙏 Could you add a comment that this min length 8 error is coming from a Cognito requirement for future us?

Maybe someday when we have all the time in the world, we could make the API handle returning nice, human-readable error messages. For now, this is the exact amount of effort that I think we should put into this. Thanks for making QA testing better! ✨